packages/chat/app/utils/errorUtils.ts

```
@@ -0,0 +1,64 @@
+
+  // Default messages based on error type
+  switch (error.type) {
+    case ErrorType.NETWORK:
+      return 'Network connection issue. Please check your internet connection and try again.';
+    case ErrorType.STORAGE:
+      return 'Unable to access storage. Your data may not be saved.';
+    case ErrorType.VALIDATION:
+      return 'Invalid input. Please check your information and try again.';
+    case ErrorType.PERMISSION:
+      return 'Permission denied. You may not have access to this feature.';
+    case ErrorType.UNKNOWN:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)